hs_swat_modelinstance/models.py

```diff
@@ -191,19 +229,27 @@ class ModelInput(AbstractMetaDataElement):
     simulation_type_choices = (('Annual', 'Annual'), ('Monthly', 'Monthly'), ('Daily', 'Daily'), ('Hourly', 'Hourly'),)
 
     warmupPeriodValue = models.CharField(max_length=100, null=True, blank=True, verbose_name='Warm-up period in years')
-    rainfallTimeStepType = models.CharField(max_length=100, choices=rainfall_type_choices, null=True, blank=True, verbose_name='Rainfall time step type')
-    rainfallTimeStepValue = models.CharField(max_length=100, null=True, blank=True, verbose_name='Rainfall time step value')
-    routingTimeStepType = models.CharField(max_length=100, choices=routing_type_choices, null=True, blank=True, verbose_name='Routing time step type')
-    routingTimeStepValue = models.CharField(max_length=100, null=True, blank=True, verbose_name='Routing time step value')
-    simulationTimeStepType = models.CharField(max_length=100,choices=simulation_type_choices, null=True, blank=True, verbose_name='Simulation time step type')
-    simulationTimeStepValue = models.CharField(max_length=100, null=True, blank=True, verbose_name='Simulation time step value')
-    watershedArea = models.CharField(max_length=100, null=True, blank=True, verbose_name='Watershed area in square kilometers')
+    rainfallTimeStepType = models.CharField(max_length=100, choices=rainfall_type_choices, null=True, blank=True,
```

Comment:
@selimnairb My preference is not to use camel casing. Since these attribute names are used in camel casing format in the xml file. I am OK to leave as it is.